### ISSUE TITLE
[libpostal] How to change language in normalize options ?

### ISSUE THREAD

**Issue by AntoineLecacheurInsee** [bytedeco/javacpp-presets#1385]
Hi,

I am trying to use your library to make libpostal work with Java since the "official library" https://github.com/openvenues/jpostal doesn't seem to be maintained at all.
I know nothing to C++ and I am struggling to understand how the presets work, and specifically how to set options for the parser and normalizer from libpostal.

I have tried the following syntax for language and country setting with the address parser, and it seems to be ok :
```java
libpostal_address_parser_options_t options = libpostal_get_address_parser_default_options();
options.language(new BytePointer("fr", "UTF-8"));
options.country(new BytePointer("fr", "UTF-8"));
```

However, I fail to understand how to use the languages setters for **libpostal_normalize_options_t**. I have tried something like this but it doesn't seem to be working :
```java
libpostal_normalize_options_t normalizeOptions = libpostal_get_default_options();
var lang = normalizeOptions.languages(new PointerPointer("fr"));
```

When I try to normalize the String "lt fine", the normalizer returns "light fine" instead of "lieutenant fine" that  I would expect.

Is it possible to provide some language setting example with the normalizer ?

Thanks !

**Comment by saudet** [comment]
If you have a working example in C++, I can help you translate it!

**Comment by AntoineLecacheurInsee** [comment]
The only example I could find might be in the libpostal tests : https://github.com/openvenues/libpostal/blob/1fe1f0af1f328487e7397560e1b45beb6200ed85/test/test_expand.c#L63
I think in this test method, options.languages are set.

I had to go check on Python examples to understand what the options where, didn't find documentation about this. I forgot to include the following information to my question :
- languages option in normalizer / expander should be a list of strings like ["fr", "en"]
- country and language options in parser are only string

So basically what I'm looking for is to pass `normalizeOptions.languages()` a list of strings

Thank you for your time 🙏

**Comment by saudet** [comment]
I'm guessing something like this should work:
```java
        options.num_languages = 2;
        options.languages = new PointerPointer("fr", "en");
```
